src/Microsoft.Azure.SignalR/ServiceOptionsExtensions.cs

```diff
@@ -14,6 +16,14 @@ public static void Validate(this ServiceOptions options)
             {
                 throw new AzureSignalRInvalidServiceOptionsException("MaxPollIntervalInSeconds", "[1,300]");
             }
+
+
+            if (!string.IsNullOrEmpty(options.ApplicationName) && !Regex.IsMatch(options.ApplicationName, "^[a-zA-Z][a-zA-Z0-9_]*$"))
```

Comment:
Add UT?

Reply:
Excuse me, what is the meaning of "Add UT"?

Reply:
Short for Unit Test.

Reply:
And you can refer to [this test case](https://github.com/Azure/azure-signalr/blob/dev/test/Microsoft.Azure.SignalR.Tests/NegotiateHandlerFacts.cs#L420) to add another fact against your new rule.